tests/integration/test_delegated_access.py

```diff
@@ -1,35 +1,56 @@
+import os
+
 import requests
 import pytest
 
+from labelbox import Client
+
 
-@pytest.mark.skip("Can only be tested in specific organizations.")
-def test_default_integration(client):
-    # This tests assumes the following:
-    # 1. gcp delegated access is configured to work with utkarsh-da-test-bucket
-    # 2. the integration name is gcp test
-    # 3. This integration is the default
+@pytest.mark.skipif(os.environ.get("DA_GCP_LABELBOX_API_KEY") is None,
+                    reason="DA_GCP_LABELBOX_API_KEY not found")
```

Comment:
Love this. When I saw the changes to the makefile I was thinking we need something like this.